{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/static/js/project.js

```diff
@@ -18,4 +18,3 @@ Issues with the above approach:
 3. Really Unforgiving: No way to override this behavior
 4. Undocumented: No mention in the documentation, or it's too hard for me to find
 */
-$('.form-group').removeClass('row');
```

Comment:
The comments above this line related to crispy forms should be removed if this line is removed.

Reply:
done. Should I stash the changes or is it fine like this?

Reply:
No, don't worry about it, we can squash from the Github UI.